### DISCUSSION TITLE
Has the barcelona template been dropped?

### DISCUSSION THREAD

**Discussion by alexroehrs** [primefaces/community#1750] [PrimeFaces Templates] [answered]
Barcelona template is discontinued?

**Comment by melloware** [comment]
IF you read through all the forum posts here about other templates there has been no official word.

**Reply by alexroehrs** [reply]
Yes, we saw that there is no official position. The problem is that, in the case of the barcelona template and perhaps others that are not available, the template is still using Jakarta 8 and, therefore, it is difficult to update a project that needs other important libraries that already use Jakarta 9 or higher. So, we would like to know if these templates will at least be updated to the new Jakarta versions or will they be updated just for security reasons. If these templates have actually been discontinued, it may be interesting to migrate to other templates that receive more updates. In short, an official position would be important to decide what to do.

**Reply by melloware** [reply]
@alexroehrs I have Barcelona 6.0.0 and it supports Jakarta EE 10 if you download it from your PrimeStore account.  Are you not seeing that?

![image](https://github.com/primefaces/community/assets/4399574/ebab5ef0-49f9-48d3-a336-b7beb4f31be0)

It has a Jakarta version of the JAR it builds for EE10.

**Reply by melloware** [reply]
I actually just helped another user update their Barcelona to support PF 13.0.8

**Reply by alexroehrs** [reply]
Version 6.0.1 does not appear to download from the PrimeStore for me. The latest versions are 5.1.0 and 6.0.0. My license type is basic. I just downloaded version 6.0.0 again and only what is in the image appears. File dates from version 6.0.0 are Nov 8, 2023.
<img width="819" alt="Screenshot 2024-04-29 at 12 03 55" src="https://github.com/primefaces/community/assets/167912549/58e8444d-61c2-48c3-aa31-165af00a0ab7">

**Reply by melloware** [reply]
yeah 6.0.1 is my local build that I patched a bug for another user but do you see the `jakarta` version of the JAR included in the zip?

**Reply by melloware** [reply]
Oh yeah I see it in your screenshot above `barcelona-theme-6.0.0-jakarta.jar`

**Reply by melloware** [reply, marked as answer]
Also see: https://github.com/orgs/primefaces/discussions/1806

**Comment by BenitoCuenca** [comment]
Please, where can I download or buy the Primefaces barcelona-theme JAR component that supports JAkarta?, for example: barcelona-theme-VERSION-jakarta.jar

Thank you very much.

**Reply by melloware** [reply]
@BenitoCuenca did you already purchase it?  The last version still should be in your PrimeStore account for download and I think it supports Jakarta.  Let me know if it doesn't.

**Reply by BenitoCuenca** [reply]
I haven't bought it yet, in the link https://www.primefaces.org/store/templates.xhtml with my PrimeStore account it doesn't appear to download or buy barcelona-theme-VERSION.jar that supports Jakarta.

Thank you very much.

**Reply by melloware** [reply]
@BenitoCuenca are you on the Discord channel if so send me a DM I can help. Discord: https://discord.gg/gzKFYnpmCY

**Comment by tandraschko** [comment]
PrimeTek decided to only maintain the following templates for now:
          - "apollo"
          - "atlantis"
          - "avalon"
          - "diamond"
          - "freya"
          - "poseidon"
          - "ultima"
          - "verona"
